File: CRUD/Php-MySQL/Classes/dbConnect.php
<?php

// Conexão com banco de dados

class Connect {

    static $connect;

    public static function getConnection() {
        if (self::$connect == null) {
            $servername = "localhost";
            $username = "root";
            $password = "12345";
            $db_name = "lanchonete";

            self::$connect = new mysqli($servername, $username, $password, $db_name);

            if (mysqli_connect_error()):
                echo "Erro na conexão: " . mysqli_connect_error();
            endif;
        }
        return self::$connect;
    }

}

File: CRUD/Java-Desktop/CodingExamples-Java-Swing-Mysql-Ex01/src/professional/DAO/DbConnect.java
package professional.DAO;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class DbConnect {

    public static Connection getConexao() {
        Connection con = null;
        try {
            String serverName = "localhost";
            String mydatabase = "LearnToImplement-JavaDesktop";
            String username = "root";
            String password = "86111411";
            String driverName = "com.mysql.jdbc.Driver";
            Class.forName(driverName);
            String url = "jdbc:mysql://" + serverName + "/" + mydatabase;
            con = DriverManager.getConnection(url, username, password);
        } catch (ClassNotFoundException e) {
            System.out.println("Driver não encontrado " + e.toString());
        } catch (SQLException e) {
            System.out.println("Erro ao conectar o Banco de Dados " + e.toString());
        }
        return con;
    }
}
